data.sql

```diff
@@ -10,4 +10,43 @@ VALUES ('Pikachu','2021-1-7',1,false,15.04);
 --INSERT 0 1
 insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg)
```

Comment:
Please add `project title` and `project description` to the `README` file. A professional README file should have complete details about the project.

Reply:
Thanks, I will do the changes